docs/src/test-configuration-js.md

```diff
@@ -174,12 +174,14 @@ export const expect = baseExpect.extend({
     }
 
     const message = pass
-      ? () => this.utils.matcherHint('toHaveAmount', locator, expected, { isNot: this.isNot }) +
+      ? () => this.utils.matcherHint('toHaveAmount', undefined, undefined, { isNot: this.isNot }) +
```

Comment:
I feel like its a good time to link to the official extend docs now so users know what `this.*` is.

https://jestjs.io/docs/expect#expectextendmatchers

Reply:
I'm not sure if we are ready to link external docs.